benchmarks/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "benchmarks",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "setup": "yarn setup:gen && yarn setup:tsc",
+    "setup:gen": "pb-gen-ts --entry-path data --ext-in-import \" \" -o data/gen simple.proto file-descriptor.proto",
```

```suggestion
    "setup:gen": "pb-gen-ts --entry-path data --ext-in-import \" \" -o data/gen",
```
you can omit entrypoint files in when using `--entry-path` option.
(you have to write that paths when using `--proto-path` option)